packages/client-react-streaming/tsup.config.ts

```diff
@@ -43,6 +43,11 @@ export default defineConfig((options) => {
       entry: {
         [output]: input,
       },
+      footer(ctx) {
+        return {
+          js: ctx.format === 'esm' ? `export const built_for_${env} = true;` : `exports.built_for_${env} = true;`
```

Comment:
I'm not sure I'm understanding the significance of the `build_for_<env>` variable here. Could you explain that?

Reply:
Sometimes the SSR and Browser bundle have the same shape, so I'm deliberately giving them different shapes by adding an additional export, so I can safely distinguish between them in the [`package-shape.json`](https://github.com/apollographql/apollo-client-nextjs/pull/263/files#diff-766f8a75e8fbcae41995751388b3269343f7729b0ec6ee34bc41926a43f8c06b) test.